terrautils/betydb.py

```diff
@@ -47,6 +48,128 @@ def get_bety_key():
                        "or create $HOME/.betykey.")
 
 
+def get_brapi_url(path=''):
+    """return brapi url from environment with optional path
+
+    Of 3 options string join, os.path.join and urlparse.urljoin, os.path.join
+    is the best at handling excessive / characters.
```

Comment:
not if you work on a windows machine in which case it will use `\` instead. I think we should use urljoin and make sure to always force path to not start with a / and the url to have a / at the end.

Reply:
changed.